pygfx/helpers/_box.py

```diff
@@ -46,27 +61,34 @@ def __init__(self, size=1.0, thickness=1, color="white"):
             dtype="f4",
         )
         positions -= 0.5
+
+        # @almarklein: does this mean we currently don't support non-isotropic scale?
+        # otherwise we may want to update this to set the tranform's scale instead.
```

Comment:
The geometry is always a cube, but the transform's scale can be used to stretch it. See `self.scale.set` below.

Reply:
In that case, why do we need `scale` _and_ `size`? Wouldn't it be enough to always construct a unit cube and use scale?

Reply:
Good point. I suspect that it has a `size` arg because many geometry-generating functions have it. And for these it makes sense, because `.scale` also sizes the children and you may not want to do that. But helper objects are unlikely to have children. I'd be ok with removing the `size` arg. @Korijn ?

Reply:
As mentioned in the other open conversation on _box.py, there is an inline geometry routine in the constructor here. That routine is parameterized so that users can generate whatever geometry they need. I guess it would make sense actually to also add subdivision parameters, since users may have UV mapping use cases for which they require more triangles on each side of the cube. But those are all concerns for a box geometry generation routine.

For the helper object, it doesn't matter too much, since indeed, you also have the standard scale/rotation/position transform options at your disposal. I ported these from three.js for the most part, and I assume the arguments are just there for one-liner visual debugging convenience. I would prefer the size arg to stay for this reason.

Reply:
@Korijn  Is it because of the name, or because of the ability to use `size` and `scale` independently? If the former, we could look into making `size` an alias of `scale` which should make our code a little less complex while maintaining the current behavior.

Reply:
I'm assuming there's no specific conscious design decision behind this.